subworkflows/nf-core/utils_nextflow_pipeline/main.nf

```diff
@@ -102,14 +98,16 @@ def checkCondaChannels() {
 
     // Check that all channels are present
     // This channel list is ordered by required channel priority.
-    def required_channels_in_order = ['conda-forge', 'bioconda']
+    def required_channels_in_order = ['conda-forge', 'bioconda', 'defaults']
```

Comment:
Aren't we removing the defaults channel?

Reply:
Don't know, but that's not the scope of this PR

Reply:
https://github.com/nf-core/modules/pull/6382
;)

Reply:
@matthdsm , I was asking because this PR seemed to reintroduce it, when it had already been removed.